### ISSUE TITLE
Error: connect ECONNREFUSED 0.0.0.0:443

### ISSUE THREAD

**Issue by tjx666** [RafaelGSS/is-my-node-vulnerable#1]
<img width="1008" alt="image" src="https://user-images.githubusercontent.com/41773861/214832503-d072848e-e4d6-4cd7-a2cf-a4447fa0201f.png">

**Comment by UlisesGascon** [comment]
yeah, this can be a DNS problem as the DNS resolution is pointing to 0.0.0.0 rather to the real Github IP.
